tests/integration/tables/prefetch.cpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+class PrefetchTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(PrefetchTest, test_sanity) {
+  QueryData const rows = execute_query(
+      "select * from prefetch where path like "
+      "'D:"
+      "\\a\\osquery\\osquery\\w\\src\\tools\\tests\\configs\\windows\\prefetch"
+      "\\%.pf'");
+  QueryData const specific_rows = execute_query(
+      "select * from prefetch where path like "
+      "'D:"
+      "\\a\\osquery\\osquery\\w\\src\\tools\\tests\\configs\\windows\\prefetch"
+      "\\%.pf' AND last_execution_time = 1620953788 AND count = 3 AND "
+      "number_of_accessed_files=53");
```

Comment:
The tests here should also work locally, not on CI only.

Normally the `tools/tests/configs` folder gets automatically copied into the build folder, and tests that need that folder path to be available, receive it via an ENV var set in CMake when running the test.
You can see an example here (TEST_CONF_FILES_DIR):
https://github.com/osquery/osquery/blob/d2be385d71f401c85872f00d479df8f499164c5a/osquery/tables/networking/CMakeLists.txt#L123-L134

So you should do the same for this test and then you can retrieve it via `osquery::getEnvVar`

Reply:
this should be fixed, it was bit more difficult/confusing than i expected.  But the tests should work on CI and locally.